src/components/layout/AppLayout.jsx

```diff
@@ -126,86 +132,91 @@ class AppLayout extends Component {
+                    onClick={retryRequiredRequests}
+                  >
+                    <Icon icon={mdiFlash} />
+                    {intl.formatMessage(messages.requiredRequestsFailed)}
+                  </InfoBar>
+                )}
+                {authRequestFailed && (
                   <InfoBar
-                    type="primary"
-                    ctaLabel={intl.formatMessage(messages.buttonReloadServices)}
-                    onClick={() => window.location.reload()}
-                    onHide={() => {
-                      this.setState({
-                        shouldShowServicesUpdatedInfoBar: false,
-                      });
-                    }}
+                    type="danger"
+                    ctaLabel="Try again"
+                    ctaLoading={areRequiredRequestsLoading}
+                    sticky
+                    onClick={retryRequiredRequests}
                   >
-                    <Icon icon={mdiPowerPlug} />
-                    {intl.formatMessage(messages.servicesUpdated)}
+                    <Icon icon={mdiFlash} />
+                    {intl.formatMessage(messages.authRequestFailed)}
                   </InfoBar>
                 )}
-              {automaticUpdates && appUpdateIsDownloaded && this.state.shouldShowAppUpdateInfoBar && (
-                <AppUpdateInfoBar
-                  onInstallUpdate={installAppUpdate}
-                  onHide={() => {
-                    this.setState({ shouldShowAppUpdateInfoBar: false });
-                  }}
-                />
-              )}
-              <BasicAuth />
-              <QuickSwitch />
-              <PublishDebugInfo />
-              {services}
-              <Outlet />
+                {automaticUpdates &&
+                  showServicesUpdatedInfoBar &&
+                  this.state.shouldShowServicesUpdatedInfoBar && (
+                    <InfoBar
+                      type="primary"
+                      ctaLabel={intl.formatMessage(
+                        messages.buttonReloadServices,
+                      )}
+                      onClick={() => window.location.reload()}
+                      onHide={() => {
+                        this.setState({
+                          shouldShowServicesUpdatedInfoBar: false,
+                        });
+                      }}
+                    >
+                      <Icon icon={mdiPowerPlug} />
+                      {intl.formatMessage(messages.servicesUpdated)}
+                    </InfoBar>
+                  )}
+                {automaticUpdates &&
+                  appUpdateIsDownloaded &&
+                  this.state.shouldShowAppUpdateInfoBar && (
+                    <AppUpdateInfoBar
+                      onInstallUpdate={installAppUpdate}
+                      updateVersionParsed={updateVersionParse(updateVersion)}
+                      onHide={() => {
+                        // eslint-disable-next-line react/no-unused-state
```

Comment:
i think this might mean some lurking issue - can the issue not be fixed without disabling eslint?

Reply:
Yep, disabling this didn't cause any issue.